hack/generate-yamls.sh

```diff
@@ -105,7 +107,7 @@ ko resolve ${KO_YAML_FLAGS} -f config/namespace-wildcard-certs | "${LABEL_YAML_C
 # Create serving.yaml with all of the default components
 cat "${SERVING_CORE_YAML}" > "${SERVING_YAML}"
 cat "${SERVING_HPA_YAML}" >> "${SERVING_YAML}"
-cat "./third_party/net-istio.yaml" >> "${SERVING_YAML}"
+cat "${NET_ISTIO_YAML}" >> "${SERVING_YAML}"
```

Comment:
Does this do `cat https://storage.googleapis.com/knative-nightly/net-istio/latest/release.yaml`? You need to download the yaml, but I cannot find where you did it.

Reply:
@nak3 Thanks for the catch. I must have downloaded in my imagination. 👍